source/common/tcp_proxy/tcp_proxy.cc

```diff
@@ -816,6 +822,11 @@ void Filter::onMaxDownstreamConnectionDuration() {
 }
 
 void Filter::onAccessLogFlushInterval() {
+  if (!getStreamInfo().streamState() ||
+      getStreamInfo().streamState() != StreamInfo::StreamState::InProgress) {
```

Comment:
Same issue here - it's not going to enable timer again on Started - is there a test that covers this situation?

Reply:
Thanks, fixed it. I added a check that reset timer is called when state is Started, but for tcp_proxy I'm currently not sure how to test it.